secret_store/src/key_server.rs

```diff
@@ -460,7 +463,7 @@ pub mod tests {
 	#[test]
 	fn decryption_session_is_delegated_when_node_does_not_have_key_share() {
 		//::logger::init_log();
```

Comment:
Not related to this PR, however this logger/comment should be enabled or remove IMHO